### ISSUE TITLE
Please can you add official support for Cocapods

### ISSUE THREAD

**Issue by grantkemp** [PaddleHQ/Paddle-MAS#11]
I just spent about an hour ripping the library out of my mac app - as no matter what I would do it says that it was generating a clang error. 

Ideally we should support Cocoapods with this by default as it generally removes the errors as the default instead of linking to the old method of copying the framework into source. 

Google officially supports Cocoapods - which seems like a good move. 

Hope you can adopt this to simplify using your libraries ( also add it to the readme doc)

**Comment by avalore** [comment]
Thanks for pointing this out @grantkemp  - we support Cocoapods in the non-MAS version of the SDK so we will add support for the MAS version of the SDK next.

**Comment by grantkemp** [comment]
Hi-  I actually saw that there is Paddle Mas on cocoapods already ( https://cocoapods.org/pods/Paddle-MAS) I managed to get it working for the feedback forms - but would be great to add the documentation (currently blank)  and link to it in your documentation.  Because the link on your site took me to the Paddle SDK - I assumed it wasn't working. 

Happy to close this -but some documentation would go a long way!
